source/extensions/network/dns_resolver/cares/dns_impl.h

```diff
@@ -128,6 +131,8 @@ class DnsResolverImpl : public DnsResolver, protected Logger::Loggable<Logger::I
   envoy::config::core::v3::DnsResolverOptions dns_resolver_options_;
 
   absl::node_hash_map<int, Event::FileEventPtr> events_;
+  const bool use_resolvers_as_fallback_;
+  std::vector<in_addr> resolvers_vector_;
```

Comment:
nit: I think this can be const

Reply:
[Passing the address](https://github.com/envoyproxy/envoy/pull/19010/files#diff-a5fe2c0400af6416558dbfc6562fa68f1c1ba5dd56c37fd2c826efe8a61a9f1bR126) of the first element to c-ares discards the const qualifier.